R/PipeOpTaskSurvClassif.R

```diff
@@ -0,0 +1,133 @@
+#' @title PipeOpTaskSurvClassif
+#' @name mlr_pipeops_trafotask_survclassif
+#' @template param_pipelines
+#'
+#' @description
+#' Transform [TaskSurv] to [TaskClassif][mlr3::TaskClassif] by (describe the method, add one ref?)
```

Comment:
describe briefly what the method does! how are the `cut` and `max_time` actually used? what is the output task? what is the long format during predict?